falcon/api.py

```diff
@@ -610,6 +613,11 @@ def set_error_serializer(self, serializer):
             property value, and it may choose to override the
             representation logic.
 
+        Note:
```

Comment:
https://github.com/falconry/falcon/blob/f078b2e6f1d54100f097b2fbb560a2b56fe7d922/falcon/api.py#L616-L619

This is not directly related to the changes in this PR, but I didn't see that this specific interaction was documented, so it seemed worth including.